OverlayPlugin.Core/PluginConfig.cs

```diff
@@ -331,21 +331,29 @@ public static PluginConfig LoadXml(string pluginDirectory, string path)
         /// <param name="pluginDirectory"></param>
         public void SetDefaultOverlayConfigs(string pluginDirectory)
         {
-            var miniparseOverlayConfig = new MiniParseOverlayConfig(DefaultMiniParseOverlayName);
-            miniparseOverlayConfig.Position = new Point(20, 20);
-            miniparseOverlayConfig.Size = new Size(500, 300);
-            miniparseOverlayConfig.Url = new Uri(Path.Combine(pluginDirectory, "resources", "miniparse.html")).ToString(); 
+            var KagerouDefaultOverlay = new MiniParseOverlayConfig("Kagerou Overlay");
```

Comment:
Modern Overlays

Reply:
please don't do this - I'll leave OverlayPlugin independent in any ways.